OpenSim/Actuators/Thelen2003Muscle.h

```diff
@@ -151,12 +138,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Thelen2003Muscle, ActivationFiberLengthMuscle);
     OpenSim_DECLARE_PROPERTY(Flen, double,
         "maximum normalized lengthening force");
 
-    //OpenSim_DECLARE_PROPERTY(activation_minimum_value, double,
-    //    "minimum activation value permitted");
-    
-    OpenSim_DECLARE_PROPERTY(fv_linear_extrap_threshold, double,
+    OpenSim_DECLARE_PROPERTY(FvExtrapolationThreshold, double,
         "fv threshold where linear extrapolation is used");
 
+    OpenSim_DECLARE_UNNAMED_PROPERTY(MuscleFirstOrderActivationDynamicModel,
```

Comment:
Is the intent to make this dependency on a base `MuscleActivationDynamicsModel` with the user having the option to choose different concrete activation models? Eventually Muscle might contain this property, with the muscle defaulting to a pass through model e.g. `a = x`.